projects/RabbitMQ.Client/client/api/ConnectionFactory.cs

```diff
@@ -355,7 +355,7 @@ public IAuthMechanismFactory AuthMechanismFactory(IList<string> mechanismNames)
         /// </exception>
         public IConnection CreateConnection()
         {
-            return CreateConnection(EndpointResolverFactory(LocalEndpoints()), ClientProvidedName);
+            return CreateConnection(ClientProvidedName);
```

Comment:
Note this is just making this overload follow the pattern in the rest of them of calling into the overload that adds a `clientProvidedName` parameter instead of directly calling the final overload.